doc/api/process.md

```diff
@@ -1554,6 +1554,18 @@ Perform a SemVer comparison to the release version.
   version, `0` if the given version matches the process version, and `-1`
   if the given version is greater than the release version.
 
+## process.release.compareVersion(version)
+<!-- YAML
+added: REPLACEME
+-->
+
+Perform a SemVer comparison to the release version.
```

Comment:
The description should come after the parameter list.

Reply:
Done (including in the original description).